tests/core/mountpoint/test_folder_operations.py

```diff
@@ -90,9 +91,11 @@ def __truediv__(self, path):
 @pytest.mark.mountpoint
 @pytest.mark.flaky(reruns=0)
 def test_folder_operations(tmpdir, caplog, hypothesis_settings, mountpoint_service_factory):
-
     tentative = 0
 
+    if IS_OXIDIZED:
+        pytest.xfail("TODO: investigate `database is locked` error")
+
```

Comment:
use [conditional parameter for `xfail`](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#condition-parameter)